modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -102,23 +66,48 @@ $ oc edit sonataflow <workflow_name> -n <your_namespace>
 +
 [source,terminal]
 ----
-$ oc get sonataflows -n <your_namespace>
+$ oc get sonataflows -n <namespace>
+----
+
+.. Find the pod of your application by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace>
 ----
 
 .. View the workflow logs by running the following command:
 +
 [source,terminal]
 ----
-$ oc logs <workflow_pod_name> -n <your_namespace>
+$ oc logs <workflow_pod_name> -n <namespace>
 ----
 
+.Testing
+
+The {ServerlessLogicOperatorName} automatically generates a route for workflow deployments using the `dev` profile. To test your workflow, you can use management console that is accessible via the created route. The management console allows you execute the workflow and inspect executed the instances in detail.
+
+. Access the management console by excuting following steps: 
+
+.. Retrieve the public URL to access the workflow service by running the following command:
++
+[source,terminal]
+----
+$ oc get route/<workflow_name> -n <namespace>
+----
++
+Note the value of `HOST/PORT` from the result of tis command, referenced as `<WORKFLOW_URL>` in next step.
+
+.. Navigate to the `<WORKFLOW_URL>/q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/workflows` in your browser. Please note that it may take some time until the management console is available. 
```

Comment:
* Don't need "please", or maybe even "note".
* "might" instead of "may" (better for translation).
```suggestion
.. Navigate to the `<WORKFLOW_URL>/q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/workflows` in your browser. It might take some time until the management console is available. 
```